src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Client/OrasClientTests.cs

```diff
@@ -47,10 +56,27 @@ private void PushOneLayerImage()
 
         private void PushMultiLayersImage()
         {
-            string command = $"push {_testOneLayerImageReference} {_baseLayerTemplatePath} {_userLayerTemplatePath}";
+            string command = $"push {_testMultiLayersImageReference} {_baseLayerTemplatePath} {_userLayerTemplatePath}";
             OrasExecution(command);
         }
 
+        [Theory]
```

Comment:
Do we have test cases for the invalid output folder while pulling? Say `GivenInvalidOutputFolder_WhenPull_ExceptionWillBeThrown`.

Reply:
Thanks, add it.